src/main/resources/application.yml

```diff
@@ -49,8 +49,8 @@ folio:
         topic-pattern: ${KAFKA_EVENTS_CONSUMER_PATTERN:(${folio.environment}\.)[a-zA-z0-9-]+\.\w+\.check-in}
         group-id: ${folio.environment}-mod-dcb-group
   system-user:
-    username: dcb-system-user
-    password: ${SYSTEM_USER_PASSWORD:dcb-system-user}
+    username: ${SYSTEM_USER_NAME:dcb-system-user}
+    password: ${SYSTEM_USER_PASSWORD}
```

Comment:
Please mention about this system environment variables in readme.md file of this project. Please refer to mod-inn-reach modules readme.md file containing the table for system environment variables.

Reply:
done